protocol.md

```diff
@@ -0,0 +1,196 @@
+Status Secure And Decentralized Messaging Protocol
+==================================================
+
+# Abstract
+
+Ethereum empowers users and developers to interact with totally new kind of applications called Dapps (Decentralized applications). These application allows to interact with the blockchain on a completely new level which is not only about exchanging values but also executing arbitrary logic. This logic can form very sophisticated programs like DAOs (Decentralized autonomous organizations). The missing part here is how users of Dapps can communicate securely and in a decentralized way with each other. Communication is an essential part of any activity. In this document, we specify a secure and decentralized messaging protocol that is capable of running on the Ethereum network.
+
+# Introduction
+
+TBD
+
+# Terminology
+
+* *Client*: a Whisper node implementing the protocol
+* *Whisper node*: an Ethereum node with Whisper V6 enabled (in the case of geth, it's `--shh` option)
+* *MailServer*: an Ethereum node with Whisper V6 enabled and a mail server registered capable of storing and providing offline messages
+* *Message*: decrypted Whisper message
+* *Envelope*: encrypted message with some metadata like topic and TTL echanged between Whisper nodes; a symmetric or assymetric key is needed to decrypt it and read the payload
+* *Offline message*: an expired envelope stored by a Whisper node permanently
+
+# Basic Assumption
+
+This protocol assumes that there is an Ethereum node that is capable of discoverying peers and implements Whisper V6 service. It also assumes that the participants of a given Whisper network accept messages with lowered PoW value.
```

Comment:
One more assumption: time is in sync for all the nodes in Status Whisper Network.